src/v/coproc/event_listener.cc

```diff
@@ -252,4 +188,38 @@ event_listener::poll_topic(model::record_batch_reader::data_t& events) {
                                  : ss::stop_iteration::no;
 };
 
+ss::future<>
+event_listener::process_events(event_batch events, model::offset last_offset) {
```

Comment:
that should probably be wrapped with gate too

Reply:
Done